src/main/java/core/basesyntax/FruitShop.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.WarehouseDaoRead;
```

Comment:
Consider renaming `WarehouseDaoRead` to `WarehouseDaoReader` for consistency with naming conventions.

Reply:
not fixed